### PR TITLE
Support reading of multiple values at once

### PR DESCRIPTION
**Pull request by golles**
- Move `kamstrup.py` to its own module
- Refactor `kamstrup.py` and add typings

@adabrandt for your code suggestion in https://github.com/bsdphk/PyKamstrup/pull/6

### CODE REVIEW

**Comment by StevenLooman** [comment]
Great work! You can consider moving kampstrup.py to its own package and publish it to PyPI even. This way others can use it even more easily. It also keeps the home assistant parts (this repo) and the kampstrup interfacing separate.

Also, parentheses are only needed when catching multiple exception types. E.g., you can omit the parentheses on this line:
https://github.com/golles/ha-kamstrup_403/pull/56/files#diff-b42daff407aa09858d366cd76d1042a12e82de32c92ea02115b6e6dd8393bae1R141

**Comment by golles** [comment]
> Great work! You can consider moving kampstrup.py to its own package and publish it to PyPI even. This way others can use it even more easily. It also keeps the home assistant parts (this repo) and the kampstrup interfacing separate.

I have thought about it, and I'm not entirely sure if I want to take public ownership of it, also because it's not really my code. But with the change I made in this PR, I agree it's pretty standalone. I'll give it another thought...

> Also, parentheses are only needed when catching multiple exception types. E.g., you can omit the parentheses on this line: https://github.com/golles/ha-kamstrup_403/pull/56/files#diff-b42daff407aa09858d366cd76d1042a12e82de32c92ea02115b6e6dd8393bae1R141

Thanks, I found a few more as well, fixed in 8192f0d96c3025b5ff06153889a73482ccf64bdd

File: custom_components/kamstrup_403/pykamstrup/kamstrup.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+            return (None, None)
+
+        if bytearray_data[0] != 0x3F or bytearray_data[1] != 0x10:
+            return (None, None)
+
+        value, unit = self._process_response(nbr, bytearray_data[2:])
+
+        return (value, unit)
+
+    def get_values(self, multiple_nbr: list[int]):
```
Comment (golles) [inline custom_components/kamstrup_403/pykamstrup/kamstrup.py:157]: - [ ] Add missing return type

**Comment by StevenLooman** [comment]
> I have thought about it, and I'm not entirely sure if I want to take public ownership of it, also because it's not really my code. But with the change I made in this PR, I agree it's pretty standalone. I'll give it another thought...

Perhaps asking @bsdphk directly helps, maybe he can give his opinion?

**Pull request merged**